### PR TITLE
IDVA3-1634 Refactor confirm company router and handler

### PR DESCRIPTION
**Pull request by AlisonGriffiths**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: src/routers/handlers/confirm-company/confirmCompany.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export class ConfirmCompanyHandler extends GenericHandler<ConfirmCompanyViewData
             backURL: companyLookup,
             company,
             address,
+            title: "Confirm this is the correct company - Provide identity verification details for a PSC or relevant legal entity",
```
Comment (PhilipGlover) [inline src/routers/handlers/confirm-company/confirmCompany.ts:41]: Why is this added here?  if there is to be a change to the page title shouldn't it be in the locales? `confirm_company_main_title`

Comment (PhilipGlover) [reply src/routers/handlers/confirm-company/confirmCompany.ts:41]: This title value is not used in the template, it retrieves it properly from locales.  I think it was added here just for the unit test, so I've removed that from the test checks

**Review by PhilipGlover — APPROVED** [review]
- have run code, all works as before
- tests pass and coverage 100% on both

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!
